pulser-core/pulser/register/_reg_drawer.py

```diff
@@ -39,15 +54,20 @@ def _draw_2D(
         blockade_radius: Optional[float] = None,
         draw_graph: bool = True,
         draw_half_radius: bool = False,
+        qubit_colors: Optional[dict[QubitId, str]] = None,
```

Comment:
This is minutia, by I guess this doesn't have to be optional, it can rather be:
```suggestion
        qubit_colors: dict[QubitId, str] = {}, 
```
It would simplify the rest of the code a bit too

Reply:
> This is minutia, by I guess this doesn't have to be optional, it can rather be:
> 
> It would simplify the rest of the code a bit too

It is generally considered as a bad practice by the community to take a mutable as a default parameter. The pitfall is that the parameter may be modified, and subsequent calls will be passed the muted parameter. Even if it's not currently modified, future modifications may not notice about it and trigger bugs.

I have found a way to circumvent this recommendation: I've added a commit where I'm declaring the parameter with `Mapping` so that mypy checks it's not written. Let me know what you think about it.

Reply:
Looks good, thanks!